### PR TITLE
feat: realworld dagger pipeline

### PR DESCRIPTION
**Pull request by yushuang200109011**
### Description

This PR enables full pipeline dagger training in the realworld, including data collection, calculate norm status, SFT, and online dagger training.

Specifically, the following things were completed:
1. Add a calculate_norm_status tool. The new dataset no longer relies on openpi to calculate norm stats.
2. Add a set of configs (including realworld dagger and sft yamls, pi0_franka_dagger policy and dataconfig) to enable direct reproducing a dual-view dagger training with a franka
3. fix realworld dagger logic
4. Modify the collect_episode wrapper and the lerobot writer to not save the non-existent image key instead of saving an empty byte. Saved empty byte lead the lerobot dataset fail to load.
5. Turn all openpi sft config's default add_value_head to False
6. Remove no_gripper param in collect_data.yaml. It's better to set it in each env config
7. Make the reward done wrapper print key press only when it is not None.
8. Add a paragraph in sft_openpi.rst to introduce calculate_norm_status tool
9. Add a full-pipeline realworld guideline in dagger.rst 
10. fix the format mistake in index.rst

### How has this been tested?

All above modifications are validated in a full-pipeline dagger training

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update (Document-only update)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: examples/embodiment/config/realworld_pnp_dagger_openpi.yaml
Comment (zanghz21) [inline examples/embodiment/config/realworld_pnp_dagger_openpi.yaml:1]: clean.

File: rlinf/models/embodiment/openpi/policies/franka_dagger_policy.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import dataclasses
+
+import einops
+import numpy as np
+import torch
+from openpi import transforms
+from openpi.models import model as _model
+
+
+def make_franka_dagger_example() -> dict:
```
Comment (zanghz21) [inline rlinf/models/embodiment/openpi/policies/franka_dagger_policy.py:23]: Is this function useful?

File: rlinf/models/embodiment/openpi/openpi_action_model.py [hunk 2]
```diff
@@ -349,26 +349,6 @@ def prepare_dagger_sft_batch(self, batch):
             obs_dict["actions"] = batch["action"].reshape(
                 bsz, self.config.action_chunk, -1
             )
-            if obs_dict["actions"].shape[2] < self.config.action_dim:
```
Comment (zanghz21) [inline rlinf/models/embodiment/openpi/openpi_action_model.py:357]: Will this part affect other components?

File: toolkits/replay_buffer/calculate_norm_stats.py
Comment (zanghz21) [inline toolkits/replay_buffer/calculate_norm_stats.py:1]: norm_stats, not status

File: rlinf/models/embodiment/openpi/policies/franka_dagger_policy.py (hunk 1 above)
Comment (yushuang200109011) [reply rlinf/models/embodiment/openpi/policies/franka_dagger_policy.py:23]: It seems useless, but every existing policy has a make_xxx_example function 🤔

File: rlinf/models/embodiment/openpi/openpi_action_model.py (hunk 2 above)
Comment (yushuang200109011) [reply rlinf/models/embodiment/openpi/openpi_action_model.py:357]: This deletion is in branch "model_action not in batch" of prepare_dagger_sft_batch, which only used in realworld dagger. It will not affect other part

**Comment by Brunch-Life** [comment]
reproduced before dagger actually run（because no enough server）
